app/controllers/api_controller/parameters.rb

```diff
@@ -66,6 +66,8 @@ def parse_filter(filter, operators)
         filter_value, method = filter_value.gsub(/^'|'$/, ''), str_method
       elsif filter_value.match(/^".*"$/)
         filter_value, method = filter_value.gsub(/^"|"$/, ''), str_method
+      elsif filter_value.match(/^NULL$|^nil$/i)
```

Comment:
this could leverage the `case` statement's `===` matching to reduce some duplication:
```ruby
case fitler_value
when /^NULL$|^nil$/i
  # ....
```
but really not necessary, just a thought!